llama-index-integrations/llms/llama-index-llms-nvidia/tests/test_text-completion.py

```diff
@@ -31,7 +30,8 @@ def __exit__(self, *exc: object) -> None:
             os.environ["NVIDIA_API_KEY"] = self.api_env_was
 
 
-def mock_completion_v1(model_name: str) -> Completion:
+def mock_completion_v1(*args, **kwargs) -> Completion:
+    model_name = kwargs.get("model")
```

Comment:
the model name doesn't appear to be important here. can you simply make it a static string?